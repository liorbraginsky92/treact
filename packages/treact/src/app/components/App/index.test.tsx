import * as React from 'react'
import * as renderer from 'react-test-renderer'
import { App } from './'

it('App renders correctly', () => {
  const tree = renderer.create(
    <App>Hello, world!</App>,
  ).toJSON()

  expect(tree).toMatchSnapshot()
})

