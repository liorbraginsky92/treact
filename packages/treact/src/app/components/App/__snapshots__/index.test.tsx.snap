// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<section
  className="css-1ehidck"
>
  Hello, world!
</section>
`;
